README.md

```diff
@@ -1,107 +1,100 @@
-
 Discovery Environment Database Schema
```

Comment:
The README updates consisted primarily of file name updates and allowing Emacs to reformat the paragraphs.